libs/designsystem/header-actions/src/header-actions.component.scss

```diff
@@ -0,0 +1,27 @@
+@use '@kirbydesign/core/src/scss/utils';
+
+:host(.right) {
+  .bounding-box {
+    flex-direction: row-reverse;
+  }
+}
+
+.bounding-box {
```

Comment:
Maybe this could just be the host element itself?

Reply:
Nice catch! I actually did just that in the follow-up PR in #2920 (which is not yet done)